server/routers/api-router.js

```diff
@@ -229,13 +229,9 @@ router.get("/api/badge/:id/uptime/:duration?", cache("5 minutes"), async (reques
     try {
         const requestedMonitorId = parseInt(request.params.id, 10);
         // if no duration is given, set value to 24 (h)
-        let requestedDuration = request.params.duration !== undefined ? request.params.duration : "24h";
+        let requestedDuration = request.params.duration !== undefined ? request.params.duration : "24";
         const overrideValue = value && parseFloat(value);
 
-        if (requestedDuration === "24") {
```

Comment:
a better check would be to append the `h` for all numeric only `requestedDuration`s

Reply:
Ohhh...Get it! So the duration could be `24`,`24h` or `30d`.Is the default unit `hours`?

Reply:
yes, the default unit should be `hours` for the badges (mostly for backwards compatability)